src/BaseHtml.php

```diff
@@ -1,95 +1,70 @@
 <?php
-/**
- * @link http://www.yiiframework.com/
- * @copyright Copyright (c) 2008 Yii Software LLC
- * @license http://www.yiiframework.com/license/
- */
+
+declare(strict_types=1);
 
 namespace Yiisoft\Yii\Bootstrap4;
 
 use Yiisoft\Arrays\ArrayHelper;
 
 /**
- * BaseHtml provides concrete implementation for [[Html]].
+ * BaseHtml provides concrete implementation for {@see Html}.
  *
- * Do not use BaseHtml. Use [[Html]] instead.
- *
- * @author Paul Klimov <klimov.paul@gmail.com>
+ * Do not use BaseHtml. Use {@see Html} instead.
  */
-class BaseHtml extends \yii\helpers\Html
+class BaseHtml extends \Yiisoft\Html\Html
```

Comment:
can be `abstract`

Reply:
I think not, since this class is the implementation of the original abstract class.

Reply:
Done.